process-automation/latest/model/connectors/teams.md

```diff
@@ -52,48 +49,48 @@ The `getTeams` action is used by the Teams connector and retrieves all of the vi
 The input parameter to retrieve all of the visible teams is:
 
 | Property | Type | Description |
-|--- | --- | --- |
+|----------|------|-------------|
 | teamName | String | *Optional.* Name of the team. If this property is null, the action will return all the teams. |
 
 The output parameter to retrieve all of the visible teams is:
 
-| Property  | Type | Description |
-|--- |--- |--- |
+| Property | Type | Description |
+|----------|------|-------------|
 | result | JSON | *Optional.* Response with the team name and the identifier returned by the Teams API. |
 
 ## Create Teams Channel
 
-The `createTeamsChannel` action is used by the Teams connector to create a new public or private channel in Teams.
+The `createTeamsChannel` action is used by the Teams connector to create a new public or privatechannel in Teams.
```

Comment:
Did you intentionally remove the gap between these two words?

Reply:
fixed.